lib/tentenone_change_notifier.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+
+class TenTenOneChangeNotifier extends ChangeNotifier {
```

Comment:
🙃 Something like that would be a bit more descriptive.

```suggestion
class StartupMessageNotifier extends ChangeNotifier {
```